vitrina/structure/models.py

```diff
@@ -542,3 +543,20 @@ def type_repr(self):
         if self.type:
             return get_type_repr(self)
         return ""
+
+
+class ValidationStatus(models.TextChoices):
+    PENDING = "pending", _("Pending")
+    VALID = "valid", _("Valid")
+    INVALID = "invalid", _("Invalid")
```

Comment:
Optional: usually but not always in katalogas we have uppercase values

```suggestion
    PENDING = "PENDING", _("Pending")
    VALID = "VALID", _("Valid")
    INVALID = "INVALID", _("Invalid")
```